src/common/Coverage.ts

```diff
@@ -36,6 +91,17 @@ export class Coverage {
     this._cssCoverage = new CSSCoverage(client);
   }
 
+  /**
+   * @param options - Set of configurable options for coverage.
+   * @param options.resetOnNavigation - Whether to reset coverage on every navigation. @defaultValue true.
```

Comment:
This doesn't seem to be the right syntax. How do we document a bag of options?

Reply:
The best fix is to extract an interface for these options so you can document them properly. Else TSDoc doesn't handle it very well :(

Reply:
Done.